cmsplugin_profile/cms_plugins.py

```diff
@@ -69,10 +69,14 @@ class ProfileGridPromoPlugin(CMSPluginBase):
         ("Featured Profiles", {
             'fields': ('profiles_field', ),
             'description':
-                _(u"{} profiles will be featured in this promo, marked by the"
-                  " border around the thumbnails. Click on a thumbnail to add"
-                  " or remove it from the group of featured profiles."
-                  .format(MAX_PROMO_PROFILES)),
+                _(u"Profile entries can be selected to be featured in a Profile"
```

Comment:
Why are we sending this through the translator? Since we interpolate the number of items allowed, this can't be translated after the fact, AFAIK.